### PR TITLE
PMM-10502 fixed FB tests

### PR DESCRIPTION
**Pull request by vyarosh**

### CODE REVIEW

**Comment by peterSirotnak** [comment]
Test run from jenkins?

**Comment by vyarosh** [comment]
https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-ui-tests/detail/pmm2-ui-tests/23113/pipeline/

File: tests/backup/locations_test.js [hunk 1]
```diff
@@ -278,7 +282,8 @@ Scenario(
     I.click(locationsPage.buttons.addLocation);
     I.verifyPopUpMessage(locationsPage.messages.successfullyEdited(updatedLocation.name));
 
-    I.seeElement(locationsPage.buttons.editByName(updatedLocation.name));
```
Comment (peterSirotnak) [inline tests/backup/locations_test.js:281]: can you use method that waits for element?

File: tests/backup/pages/restorePage.js [hunk 2]
```diff
@@ -17,6 +18,6 @@ module.exports = {
   waitForRestoreSuccess(backupName) {
     I.amOnPage(this.url);
     I.waitForVisible(this.elements.backupStatusByName(backupName), 180);
-    I.waitForText('Success', 30, this.elements.backupStatusByName(backupName));
```
Comment (peterSirotnak) [inline tests/backup/pages/restorePage.js:20]: same issue, can you wait for element before checking attributes.

File: tests/backup/pages/scheduledPage.js [hunk 3]
```diff
@@ -103,8 +120,8 @@ module.exports = {
   },
 
   verifyBackupDetailsRow(name, description, dataModel, cronExpression) {
-    I.seeElement(this.elements.scheduleName(name));
-    I.click(this.elements.scheduleName(name));
+    I.seeElement(this.buttons.showDetails(name));
```
Comment (peterSirotnak) [inline tests/backup/pages/scheduledPage.js:123]: wait for element.

File: tests/backup/scheduled_test.js [hunk 4]
```diff
@@ -336,18 +338,15 @@ Scenario('PMM-T901 Verify user can delete scheduled backup @backup @bm-mongo',
...
-    // Open Delete modal
-    I.click(scheduledPage.buttons.deleteByName(schedule.name));
+    scheduledPage.openDeleteModal(schedule.name);
 
     // Click Cancel button and verify schedule still exists
-    I.waitForVisible(scheduledPage.buttons.confirmDelete, 10);
     I.click(scheduledPage.buttons.cancelDelete);
     I.dontSeeElement(scheduledPage.elements.modalContent);
-    I.seeElement(scheduledPage.buttons.deleteByName(schedule.name));
+    I.seeElement(scheduledPage.buttons.actionsMenuByName(schedule.name));
```
Comment (peterSirotnak) [inline tests/backup/scheduled_test.js:346]: waitForVisible

File: tests/backup/inventory_test.js [hunk 5]
```diff
@@ -259,7 +259,7 @@ Scenario(
     I.fillField(backupInventoryPage.fields.description, 'Test description');
     I.click(backupInventoryPage.buttons.addBackup);
     backupInventoryPage.verifyBackupSucceeded(backupName);
-    I.seeCssPropertiesOnElements(backupInventoryPage.elements.backupNameSpan(backupName), { 'text-overflow': 'ellipsis' });
```
Comment (peterSirotnak) [inline tests/backup/inventory_test.js:262]: waitForVisible first

File: tests/backup/pages/scheduledPage.js (hunk 3 above)
Comment (vyarosh) [reply tests/backup/pages/scheduledPage.js:123]: wait is inside the previous method in line #122

File: tests/backup/pages/restorePage.js (hunk 2 above)
Comment (vyarosh) [reply tests/backup/pages/restorePage.js:20]: the comment is for the removed line, wait is a line earlier - line 20 wait, line 21 - check

File: tests/backup/inventory_test.js (hunk 5 above)
Comment (vyarosh) [reply tests/backup/inventory_test.js:262]: the comment is for the removed line, the wait is in method in line 261

File: tests/backup/pages/scheduledPage.js (hunk 3 above)
Comment (peterSirotnak) [reply tests/backup/pages/scheduledPage.js:123]: Line 122 is declaration of method, 
`  verifyBackupDetailsRow(name, description, dataModel, cronExpression) {`

File: tests/backup/scheduled_test.js (hunk 4 above)
Comment (vyarosh) [reply tests/backup/scheduled_test.js:346]: added. though for this very line wait is not required - it's reasonable for the previous check.

**Comment by vyarosh** [comment]
https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-ui-tests/detail/pmm2-ui-tests/23650/pipeline/

**Review by peterSirotnak — APPROVED** [review]

**Pull request merged**